### ISSUE TITLE
 warning C4005: APIENTRY: изменение макроопределения

### ISSUE THREAD

**Issue by UnrealKaraulov** [glfw/glfw#2026]
2>C:\Program Files (x86)\Windows Kits\10\Include\10.0.19041.0\shared\minwindef.h(130,1): warning C4005: APIENTRY: изменение макроопределения (компилируется исходный файл .\..\src\main.cpp)
2>.\..\glfw\include\GLFW/glfw3.h(124): message : см. предыдущее определение "APIENTRY" (компилируется исходный файл .\..\src\main.cpp)

**Comment by elmindreda** [comment]
Yep, this is an annoying issue. Because so many OpenGL headers need `APIENTRY` defined, the GLFW header leaves it defined unless it has already been defined earlier.

You should be able to fix this by switching the order of includes in `main.cpp`. The GLFW header needs to be included after any Windows SDK headers and any header that includes a Windows SDK header.

**Comment by UnrealKaraulov** [comment]
@elmindreda or need remove APIENTRY from glfw, then works without problem. Because in WINAPI this is already defined...
